From 1e6ffc6294851a33547265fa12466dbdf180a621 Mon Sep 17 00:00:00 2001
From: Sam Lantinga <slouken@libsdl.org>
Date: Thu, 9 Mar 2023 16:11:53 -0800
Subject: [PATCH 288/458] Removed a duplicate delay accidentally added during
 code refactor

(cherry picked from commit 926db1bd544364a9e4ab61e60bcfbe75e05adf56)
---
 src/joystick/hidapi/SDL_hidapijoystick.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/joystick/hidapi/SDL_hidapijoystick.c b/src/joystick/hidapi/SDL_hidapijoystick.c
index a72f44699..3045f3bd7 100644
--- a/src/joystick/hidapi/SDL_hidapijoystick.c
+++ b/src/joystick/hidapi/SDL_hidapijoystick.c
@@ -448,9 +448,6 @@ static void HIDAPI_SetupDeviceDriver(SDL_HIDAPI_Device *device, SDL_bool *remove
                 int lock_count = 0;
                 char *path = SDL_strdup(device->path);
 
-                /* Wait a little bit for the device to initialize */
-                SDL_Delay(10);
-
                 SDL_AssertJoysticksLocked();
                 while (SDL_JoysticksLocked()) {
                     ++lock_count;
-- 
2.17.1

